api/src/main/java/pl/touk/nussknacker/engine/api/AdditionalVariable.java

```diff
@@ -12,4 +12,6 @@
     String name();
 
     Class clazz();
+
+    boolean initializeValue() default false;
```

Comment:
by `initialize value` you mean that initialization of value will be done by Nussknacker or user should initialize value in expression? maybe a little bit more descriptive name showing who "initialize"? Maybe "initializedByRuntime"?

Reply:
:+1: done